static/app/views/explore/logs/types.tsx

```diff
@@ -25,14 +25,19 @@ export enum OurLogKnownFieldKey {
   PROJECT = 'project',
   SPAN_ID = 'span_id',
   TIMESTAMP = 'timestamp',
-  TIMESTAMP_PRECISE = 'tags[sentry.timestamp_precise,number]',
-  OBSERVED_TIMESTAMP_PRECISE = 'sentry.observed_timestamp_nanos',
+  TIMESTAMP_PRECISE = 'timestamp_precise',
+  OBSERVED_TIMESTAMP_PRECISE = 'observed_timestamp',
+
+  PAYLOAD_SIZE = 'tags[sentry.payload_size_bytes,number]', // payload_size
```

Comment:
should this refer to the alias you gave it instead?

Reply:
It will, I was just trying again before changing it